usecases/sai-ptf/ptf-conftest.py

```diff
@@ -0,0 +1,52 @@
+import sys
+import pytest
+from saichallenger.common.sai_testbed import SaiTestbedMeta
+
+
+def import_base_modules():
+    sys.path.insert(0, '/sai-challenger/ptf/src')
+
+import_base_modules()
+
+
+@pytest.fixture(scope="session", autouse=True)
+def set_ptf_params(request):
+    if request.config.option.testbed:
+        tb_params = SaiTestbedMeta("/sai-challenger", request.config.option.testbed)
+        ports = to_ptf_int_list(tb_params.config['dataplane'][0]['port_groups'])
+    else:
+        ports = ""
+    
+    arg_back = sys.argv
+    # provide required PTF runner params to avoid exiting with an error
+    sys.argv = ['ptf.py','--test-dir', '/sai-challenger/sai/ptf', *ports]
```

Comment:
should be /sai-challenger/usecases/sai-ptf/SAI/ptf ?

Reply:
yeh... thanks